README.md

```diff
@@ -14,7 +14,7 @@ steps:
     plugins:
       - automattic/git-s3-cache#v1.0.0:
           bucket: "my-s3-bucket"
-          repo: "path-to-repo-in-s3/" # note the trailing slash – it must be present
+          repo: "path-to-repo-in-s3/"
```

Comment:
Not true anymore, now that https://github.com/Automattic/git-s3-cache-buildkite-plugin/pull/6 landed.